docs/database/elasticsearch.md

```diff
@@ -95,7 +95,7 @@ Tracing instrumentations that do so, MUST also set `http.request.method_original
 
 | Key                                 | Value                                                                                                                               |
 |:------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------|
-| Span name                           | `"search"`                                                                                                                          |
+| Span name                           | `"search elasticsearch.mydomain.com:9200"`                                                                                          |
```

Comment:
I'd suggest not to change the span name for the default case for Elasticsearch. `search` is the name of the endpoint id, so that is a fairly meaningful span name (it indicates that this was a standard sync search query). The [current spec for Elasticsearch already defines this](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/database/elasticsearch.md#span-name) - so I'd keep it.

Few other example span name from the current spec: `async_search.get`, `async_search.delete` `bulk`. I think it's worth deviating from the general span name scheme in case of Elasticsearch.

Reply:
thanks for pointing it out. I don't mind giving individual semconv more freedom to define how the span name should look like.

I wonder what's the motivation not to include `server.address:port` in the span name and whether we should remove it from the generic fallback values then?